frontend/src/scenes/notebooks/Nodes/NotebookNodeExperiment.tsx

```diff
@@ -0,0 +1,145 @@
+                                <div className="p-2">
+                                    {/* TODO: Preview is currently shared between all experiments, so if there are 2 experiments in a notebook,
+                                they will both show the same preview. This is because there is a single experiment insight ID only, i.e. `EXPERIMENT_INSIGHT_ID` */}
+                                    <ExperimentPreview
+                                        experimentId={id}
+                                        trendCount={trendCount}
+                                        trendExposure={experiment?.parameters.recommended_running_time}
+                                        funnelSampleSize={experiment?.parameters.recommended_sample_size}
+                                        funnelEntrants={entrants}
+                                        funnelConversionRate={conversionRate}
+                                    />
+                                </div>
+                            </>
+                        )}
+                        {id && isExperimentRunning && (
+                            <>
+                                {/* show results when the experiment is running */}
+                                <LemonDivider className="my-0" />
+                                <div className="p-2">
+                                    <ExperimentResult />
+                                </div>
+                            </>
+                        )}
+                    </>
+                ) : null}
+
+                <LemonDivider className="my-0" />
+                <div className="p-2 mr-1 flex justify-end gap-2">
+                    <LemonButton
+                        type="secondary"
+                        size="small"
+                        icon={<IconFlag />}
+                        onClick={() => {
+                            insertAfter(buildFlagContent(experiment.feature_flag?.id || 'new'))
+                        }}
+                    >
+                        View Feature Flag
+                    </LemonButton>
+                    {/* <LemonButton
+                        onClick={() => {
+                            insertAfter(buildPlaylistContent(recordingFilterForFlag))
+                        }}
+                        type="secondary"
+                        size="small"
+                        icon={<IconRecording />}
+                    >
+                        View Replays
+                    </LemonButton> */}
+                </div>
+            </BindLogic>
+        </div>
+    )
+}
+
+type NotebookNodeExperimentAttributes = {
+    id: ExperimentLogicProps['experimentId']
+}
+
+export const NotebookNodeExperiment = createPostHogWidgetNode<NotebookNodeExperimentAttributes>({
+    nodeType: NotebookNodeType.Experiment,
+    title: 'Experiment',
+    Component,
+    heightEstimate: '3rem',
+    href: (attrs) => urls.experiment(attrs.id ?? 'new'),
```

Comment:
Could you also do `urls.experiment(Number(attrs.id)) `? Given I assume you can't create experiments in notebooks yet

Reply:
yeah the `id` typing is a bit broken, I'll change that to always be defined for now, since not dealing with the creation flow yet.